fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java

```diff
@@ -208,12 +208,6 @@ private CacheMode innerCheckCacheMode(long now) {
                 LOG.debug("query contains non-olap table. queryid {}", DebugUtil.printId(queryId));
                 return CacheMode.None;
             }
-            if (enablePartitionCache() && ((OlapScanNode) node).getSelectedPartitionNum() > 1
```

Comment:
we should better figure out why we add these codes before, the codes added from https://github.com/apache/doris/pull/10319